### PR TITLE
Fix cli release upload to github

### LINKED ISSUE #58: fix cli release upload to github
In #51 we had an issue related to Save upload to github. Need to fix it, so everything will be automated

### PR DESCRIPTION
**Pull request by Cheshiriks**
What's done:
* add linkReleaseExecutableMultiplatform task

close #58

### CODE REVIEW

File: save-cli/build.gradle.kts [hunk 1]
```diff
@@ -54,6 +54,14 @@ kotlin {
             }
         }
     }
+
+    project.tasks.register("linkReleaseExecutableMultiplatform") {
+        when (saveTarget.name) {
+            "linuxX64" -> dependsOn(tasks.getByName("linkReleaseExecutableLinuxX64"))
+            "mingwX64" -> dependsOn(tasks.getByName("linkReleaseExecutableMingwX64"))
+            "macosX64" -> dependsOn(tasks.getByName("linkReleaseExecutableMacosX64"))
+        }
```
Comment (petertrr) [inline save-cli/build.gradle.kts:63]:
```suggestion
        dependsOn(tasks.getByName("linkReleaseExecutable${saveTarget.name.capitalize()}"))
```
Maybe, like this?

File: save-core/build.gradle.kts [hunk 2]
```diff
@@ -16,15 +15,11 @@ kotlin {
             }
         }
     }
-    val os = getCurrentOperatingSystem()
-    // Create a target for the host platform.
-    val hostTarget = when {
```
Comment (petertrr) [inline save-core/build.gradle.kts:21]: This should be changed for plugins as well, it affects all artifacts that we are deploying

**Review by petertrr — CHANGES_REQUESTED** [review]

File: save-core/build.gradle.kts (hunk 2 above)
Comment (Cheshiriks) [reply save-core/build.gradle.kts:21]: Fixed

Comment (petertrr) [reply save-core/build.gradle.kts:21]: And `warn-plugin`?

Comment (Cheshiriks) [reply save-core/build.gradle.kts:21]: Done

**Review by petertrr — COMMENTED** [review]
Maybe, action `download-artifact` creates a directory for artifacts, and because of that `upload-asset` skips them?

**Review by petertrr — COMMENTED** [review]
We should also add a constraint to release workflow: start build only if git working tree is clean. It can be done in a separate issue, though

**Comment by Cheshiriks** [comment]
> Maybe, action `download-artifact` creates a directory for artifacts, and because of that `upload-asset` skips them?

But we ourselves create a directory for loading artifacts in _download-artifact_ - tmpFolder. And then we try to upload them from it in _upload-release-asset_.

**Review by petertrr — APPROVED** [review]
lgtm

**Pull request merged**